### DISCUSSION TITLE
Since I didn't express it well before, I created a new post to describe my problem. Here's an example in sofa where I changed template="Vec3d" to template="CudaVec3f" to improve the FPS, but it got the following error:

### DISCUSSION THREAD

**Discussion by yangwxb** [sofa-framework/sofa#4692] [SofaCUDA]
![1](https://github.com/sofa-framework/sofa/assets/167422897/6c074f27-b58b-46da-a572-73cc57c0dc7a)
![2](https://github.com/sofa-framework/sofa/assets/167422897/7f395b15-0828-42bb-af5a-ad5f3f3d0443)

**Comment by bakpaul** [comment]
You didn't add any informatio in this post. I'll close it so we can continue on the previous one.
